### DISCUSSION TITLE
ordenar_tres_valores

### DISCUSSION THREAD

**Discussion by ghost** [community/community#44644] [New to GitHub] [answered]
/*Escribir un algoritmo que lea tres
  valores enteros (A, B y C) en
 teclado.  Ordenar valores A, B
  y C por intercambios sucesivos de
 manera de obtener:
 valor(A) valor(B) valor(C)
 Muestre los tres valores.
 */
 Algoritmo sort_tres_valor
 Variable
 A, B, C: entero
 temperatura: entero
 Inicio
 escribir ("Ingrese A:")
 leer un)
 escribir ("Ingrese B:")
 leer (B)
 escribir ("Ingrese C:")
 leer
 si (B < A) entonces
 temperatura <-A
 A<-B
 B <- temperatura
 terminara si
 si (C < A) entonces
 temperatura <- A
 A<-C
 C <- temperatura
 terminara si
 si (C < B) entonces
 temperatura <-B
 B<-C
 C <- temperatura
 terminara si
 escribir (A, " ", B, " ", C)
 Fin

**Comment by queenofcorgis** [comment, marked as answer]
Hi @AlexisDisz ,
The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was locked and will be removed because it violated our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. Posting screenshots of test questions without context will likely discourage engagement and responses.

If this content is posted again, your Community account will be subject to immediate and indefinite suspension.
For additional guidance, please review our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service).

Thank you!
